nym-wallet/src-tauri/src/wallet_storage/mod.rs

```diff
@@ -138,18 +138,40 @@ fn store_login_at_file(
     write_to_file(filepath, &stored_wallet)
 }
 
-pub(crate) fn store_login_with_multiple_accounts(
+/// Store the login with multiple accounts support
+///
+/// Set `overwrite` to true to overwrite the wallet file in case of password update
+pub(crate) fn save_login(
```

Comment:
Imho, with overwrite = false you are effectively appending a login to a file, with overwrite = true you're not appending, but resaving existing logins already in the file. So it feels like the latter should be a separate function. Basically the first is a `.append()` and the latter is a `.for_each(mutate)`.

```
fn store_login_with_multiple_accounts()

fn re_store_all_logins()
```